src/Protocol.cpp

```diff
@@ -0,0 +1,156 @@
+#include <cstdlib>
+#include <sonar_oculus_m750d/Protocol.hpp>
+#include <string.h>
+
+using namespace sonar_oculus_m750d;
+
+void Protocol::handleBuffer(uint8_t const* buffer)
```

Comment:
This should return a flag saying what got updated, to allow the API client to know whether e.g. there is a new sonar scan to read. We will want to handle configuration at some point, which will break this implementation.